announcements/2023hackwasm.md

```diff
@@ -0,0 +1,19 @@
+# Mission
+Create a distinctive DeFi-oriented DApp (if possible) by leveraging our latest feature, the Dynamic Link, within the CosmWasm ecosystem designed to operate on the Finschia testnet.
+
+Finschia, a hub chain powered by the Ostracon consensus engine, presents an ideal platform for the deployment of diverse DApps. While we have witnessed the successful implementation of numerous NFT-related DApps on our chain in the past, we are particularly excited about the potential emergence of innovative DeFi-oriented DApps during this hackathon, which will undoubtedly broaden our horizons.
+
+For this challenge, participants are encouraged to develop a CosmWasm contract. While there are no specific limitations, incorporating advanced features such as the Dynamic Link for inter-contract communication or utilizing the IBC protocol to enable cross-chain DApp functionality will earn additional points. We eagerly await the development of exceptional DeFi-oriented DApps across various domains, including but not limited to Payment, Lending, Decentralized Exchanges (DEX), Liquid Staking, Insurance, Bonds, Credit Rating, Asset Management, Risk Assessment/Management, and Asset Tokenization.
```

Comment:
I think  it's nice to add link in `Dynamic Link` if we complete `Dynamic Link` tutorial.